### PR TITLE
feat: title item list retry

### PR DESCRIPTION
**Pull request by rbadici-plenty**
## Why:

Closes: #ID

## Checklist before requesting a review
- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,3 +0,0 @@
-nodeLinker: node-modules
```
Comment (maxiroellplenty) [inline .yarnrc.yml:1]: dont delete this file

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

File: .yarnrc.yml (hunk 1 above)
Comment (ksted) [reply .yarnrc.yml:1]: @maxiroellplenty It's fine to delete it, we don't need it anymore for the workflows. But if you do, please also remove it from the git index via `git rm --cached .yarnrc.yml`.

Comment (maxiroellplenty) [reply .yarnrc.yml:1]: ah ok, I see 👍

File: apps/web/components/RecommendedProducts/RecommendedProducts.vue [hunk 2]
```diff
@@ -1,7 +1,4 @@
 <template>
-  <p data-testid="recommended-products" class="my-4 typography-text-lg">
```
Comment (maxiroellplenty) [inline apps/web/components/RecommendedProducts/RecommendedProducts.vue:2]: Add this text to the product page above the Recommended products

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

**Review by maxiroellplenty — APPROVED** [review]

File: apps/web/pages/index.vue [hunk 3]
```diff
@@ -106,6 +106,7 @@
     </div>
     <section class="mx-4 mt-28 mb-20 overflow-hidden">
       <NuxtLazyHydrate when-visible>
+        <p data-testid="recommended-products" class="my-4 typography-text-lg">{{ $t('moreItemsOfThisCategory') }}</p>
```
Comment (maxiroellplenty) [inline apps/web/pages/index.vue]: only show this text if there are recommended products

**Review by maxiroellplenty — CHANGES_REQUESTED** [review]

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**